### PR TITLE
2-Regression/translations/README.es.md

### PR DESCRIPTION
**Pull request by davidit33**

### CODE REVIEW

File: 2-Regression/translations/README.es.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+> Foto de <a href="https://unsplash.com/@teutschmann?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Beth Teutschmann</a> en <a href="https://unsplash.com/s/photos/jack-o-lanterns?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+  
+## Lo que vas a aprender
+
+Las lecciones de esta sección cubren los tipos de regressión en el contexto de machine learning. Los modelos de regresión pueden ayudar a determinar la _relación_ entre variables. Este tipo de modelos puede predecir valores como la longitud, la temperatura o la edad, descubriendo así relaciones entre variables a medida que analiza puntos de datos.
+
+En esta serie de lecciones, descubrirá la diferencia entre la regresión lineal y la logística, y cuándo debe usar una u otra.
+
+En este grupo de lecciones, se preparará para comenzar las tares de machine learning, incluida la configuración de Visual Studio code para manejar los cuadernos, el entorno común para los científicos de datos. Descubrirá Scikit-learn, una librería para machine learning, y creará sus primeros modelos, centrándose en los modelos de Regresión en este capítulo.
```
Comment (alfredodeza) [inline 2-Regression/translations/README.es.md:15]:
```suggestion
En este grupo de lecciones, se preparará para comenzar las tares de machine learning, incluida la configuración de Visual Studio Code para manejar los cuadernos, el entorno común para los científicos de datos. Descubrirá Scikit-learn, una librería para machine learning, y creará sus primeros modelos, centrándose en los modelos de Regresión en este capítulo.
```

File: 2-Regression/translations/README.es.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+> Existen herramientas útiles low-code que pueden ayudarlo a aprender a trabjar con modelos de regresión. Pruebe [Azure ML for this task](https://docs.microsoft.com/learn/modules/create-regression-model-azure-machine-learning-designer/?WT.mc_id=academic-15963-cxa)
```
Comment (alfredodeza) [inline 2-Regression/translations/README.es.md:17]:
```suggestion
> Existen herramientas útiles _low-code_ que pueden ayudarlo a aprender a trabjar con modelos de regresión. Pruebe [Azure ML para esta tarea](https://docs.microsoft.com/learn/modules/create-regression-model-azure-machine-learning-designer/?WT.mc_id=academic-15963-cxa)
```

File: 2-Regression/translations/README.es.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+4. [Regresión logística](4-Logistic/README.md)
+
+---
+### Créditos
+
+"ML con regresión" fue escrito con ♥️ por [Jen Looper](https://twitter.com/jenlooper)
+
+♥️ Los contribuyentes del cuestionario incluyen: [Muhammad Sakib Khan Inan](https://twitter.com/Sakibinan) y [Ornella Altunyan](https://twitter.com/ornelladotcom)
+
+El dateset de calabaza es sugerido por [este proyecto en Kaggle](https://www.kaggle.com/usda/a-year-of-pumpkin-prices) y sus datos provienen de [Specialty Crops Terminal Markets Standard Reports](https://www.marketnews.usda.gov/mnp/fv-report-config-step1?type=termPrice) distribuido por el Departamento de Agricultura de los Estados Unidos. Hemos agragado algunos puntos alrededor color basados en la variedad para normalizar la distribución. Estos datos son de dominio público.
```
Comment (alfredodeza) [inline 2-Regression/translations/README.es.md:33]:
```suggestion
El _dataset_ de calabaza es sugerido por [este proyecto en Kaggle](https://www.kaggle.com/usda/a-year-of-pumpkin-prices) y sus datos provienen de [Specialty Crops Terminal Markets Standard Reports](https://www.marketnews.usda.gov/mnp/fv-report-config-step1?type=termPrice) distribuido por el Departamento de Agricultura de los Estados Unidos. Hemos agragado algunos puntos alrededor color basados en la variedad para normalizar la distribución. Estos datos son de dominio público.
```

**Review by alfredodeza — CHANGES_REQUESTED** [review]
It looks like you have a merge commit, if possible, try to get rid of those with a rebase.

**Comment by davidit33** [comment]
> It looks like you have a merge commit, if possible, try to get rid of those with a rebase.

Thanks!

**Comment by jlooper** [comment]
ready to merge?

**Comment by davidit33** [comment]
> ready to merge?

Yes.

**Pull request merged**